src/systems/userinput/resolve-action-sets.js

```diff
@@ -139,18 +137,86 @@ export function resolveActionSets() {
       rightRemote.held.components.tags.data.isPen
   );
   userinput.toggleSet(
-    sets.cursorHoldingCamera,
+    sets.rightCursorHoldingCamera,
     !rightHand.held && !rightHand.hovered && rightRemote.held && rightRemote.held.components["camera-tool"]
   );
   userinput.toggleSet(
-    sets.cursorHoldingUI,
+    sets.rightCursorHoldingUI,
     !rightHand.held &&
       !rightHand.hovered &&
       rightRemote.held &&
       rightRemote.held.components.tags &&
       rightRemote.held.components.tags.data.holdableButton
   );
-  userinput.toggleSet(sets.cursorHoldingInteractable, rightRemote.held);
+  userinput.toggleSet(sets.rightCursorHoldingInteractable, rightRemote.held);
```

Comment:
Since this looks like a 1:1 copy, might be nice to DRY it up otherwise I guarantee we will accidentally diverge it in the future.